File: Data Structures & Algorithms/two-integer-sum/submission-2.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int i = 0; i < nums.length(); i++) {
            for(int j = 1; i < nums.length(); i++) {
                if(nums.getValue(i) + nums.getValue(j) == target) {
                    return("[" + i + "," + j + "]");
                }
            }
        }
    }
}

File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-2.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        left, right = 0, len(nums) - 1

        while left < right:
            mid = left + ((right - left) // 2)

            if nums[left] < target:
                left = mid + 1
            elif nums[right] > target:
                right = mid
            else:
                return nums[left]
        return -1